public/app/features/alerting/unified/mocks/server/handlers.ts

```diff
@@ -4,7 +4,60 @@
 
 import { HttpResponse, http } from 'msw';
 
+import { mockAlertmanagerAlert, mockSilences } from 'app/features/alerting/unified/mocks';
 import { defaultAlertmanagerChoiceResponse } from 'app/features/alerting/unified/mocks/alertmanagerApi';
+import { AlertState } from 'app/plugins/datasource/alertmanager/types';
+
+///////////////////
+// Alertmanagers //
+///////////////////
 
 export const alertmanagerChoiceHandler = (response = defaultAlertmanagerChoiceResponse) =>
   http.get('/api/v1/ngalert', () => HttpResponse.json(response));
+
+const alertmanagerAlertsListHandler = () =>
+  http.get('/api/alertmanager/:datasourceUid/api/v2/alerts', () =>
+    HttpResponse.json([
+      mockAlertmanagerAlert({
+        labels: { foo: 'bar', buzz: 'bazz' },
+        status: { state: AlertState.Suppressed, silencedBy: ['12345'], inhibitedBy: [] },
+      }),
+      mockAlertmanagerAlert({
+        labels: { foo: 'bar', buzz: 'bazz' },
+        status: { state: AlertState.Suppressed, silencedBy: ['12345'], inhibitedBy: [] },
+      }),
+    ])
+  );
+
+/////////////////
+// Datasources //
+/////////////////
+
+// TODO: Add more accurate endpoint responses as tests require
+const datasourceBuildInfoHandler = () =>
+  http.get('/api/datasources/proxy/uid/:datasourceUid/api/v1/status/buildinfo', () => HttpResponse.json({}));
```

Comment:
Likewise, not sure what we want to return here as when I followed the logic through on my local it was always 404'ing but not caring, so I don't know what we care about in particular here.

Reply:
In the best scenario this should be tied to datasources setup.
404, at least in theory should be pretty rare (only Cortex responds this way)
For Prometheus and Mimir datasources we should provide valid JSON. GMA doesn't send this request

Reply:
What would you suggest as a change in this PR? Or could we start generalising this as we move other tests to the consolidated MSW server?

Reply:
Let's postpone the decision. Not an issue in this PR